internal/pkg/cli/app_init.go

```
@@ -232,6 +234,14 @@ func (o *initAppOpts) validateAppName(name string) error {
 	if err != nil {
 		var noSuchAppErr *config.ErrNoSuchApplication
 		if errors.As(err, &noSuchAppErr) {
+			roleName := fmt.Sprintf("%s-adminrole", name)
+			tags, err := o.iamRoleManager.ListRoleTags(roleName)
+			if err != nil {
+				return nil
```

nit: we can probably add a comment here stating why we swallow the error.